tango/sweeps.py

```diff
@@ -0,0 +1,68 @@
+import itertools
+import json
+import subprocess
+from collections import OrderedDict
+from dataclasses import dataclass
+
+from tango.common import Params, Registrable
+
+
+class Sweeper(Registrable):
+    def __init__(
+        self,
+        main_config_path: str,
+        sweeps_config_path: str,
+        components: str,
+    ):
+        super(Registrable, self).__init__()
+        self.main_config_path = main_config_path
+        self.sweep_config = load_config(sweeps_config_path)
+        self.main_config_path = main_config_path
+        self.components = components
+
+    # returns all the combinations of hyperparameters in the form of a list of lists
+    def get_combinations(self) -> list:
```

Comment:
For compat with older Python versions:
```suggestion
    def get_combinations(self) -> List:
```